gtk3/lib/gtk3/icon-view.rb

```diff
@@ -15,8 +15,20 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
 
 module Gtk
-  class IconView
-    alias_method :get_cell_rect_raw, :get_cell_rect
+  class IconView 
+    alias_method :initialize_raw, :initialize, :get_cell_rect_raw, :get_cell_rect
```

Comment:
Two `alias_method`s should not be merged.
Please don't touch `alias_method :get_cell_rect_raw, :get_cell_rect`.